src/ConfigurableFirmata.h

```diff
@@ -93,6 +93,11 @@
 #define SYSEX_NON_REALTIME      0x7E // MIDI Reserved for non-realtime messages
 #define SYSEX_REALTIME          0x7F // MIDI Reserved for realtime messages
 
+// DeviceFirmata commands
+
+#define DEVICE_QUERY            0x30 // message requesting action from a device driver (DeviceFirmata)
```

Comment:
That's fine, no problem.  I just picked them to be out of the way of  
anything else that might be developing concurrently.

On Sun, 19 Jun 2016 23:05:52 -0700, Jeff Hoefs <notifications@github.com>  
wrote:

>
> In src/ConfigurableFirmata.h:
>> @@ -93,6 +93,11 @@
>>  #define SYSEX_NON_REALTIME      0x7E // MIDI Reserved for non-realtime  
>> messages
>>  #define SYSEX_REALTIME          0x7F // MIDI Reserved for realtime  
>> messages
>>  +// DeviceFirmata commands
>> +
>> +#define DEVICE_QUERY            0x30 // message requesting action from  
>> a device driver (DeviceFirmata)
>
> I'd prefer if the command IDs for DEVICE_QUERY and DEVICE_RESPONSE were  
> closer to the existing allocations, so perhaps 0x5E and 0x5F rather than  
> >0x30 and 0x31. This is in case it's ever necessary to reserve a block  
> of command IDs at some point in the future it keeps a wider range open.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.



-- 
Using Opera's mail client: http://www.opera.com/mail/